lib/KLogo/__tests__/KLogo.spec.js

```diff
@@ -0,0 +1,62 @@
+import { renderComponentForVisualTest, takeSnapshot } from '../../../jest.conf/visual.testUtils';
+
+describe.visual('KLogo', () => {
+  const snapshotOptions = { widths: [400], minHeight: 512 };
+  describe('Render KLogo with different props', () => {
+    it('Render a default logo', async () => {
+      await renderComponentForVisualTest('KLogo', { altText: 'Kolibri default logo', size: '150' });
+      await takeSnapshot('KLogo - default', snapshotOptions);
+    });
+    it('Render a logo with background', async () => {
+      await renderComponentForVisualTest('KLogo', {
+        altText: 'Kolibri Logo with background',
+        size: '150',
+        showBackground: true,
+      });
+      await takeSnapshot('KLogo - background', snapshotOptions);
+    });
+    it('Render a logo with rectangular background', async () => {
+      await renderComponentForVisualTest('KLogo', {
+        altText: 'Kolibri rectangular background',
+        size: '150',
+        backgroundStyle: 'rect',
+        showBackground: true,
+      });
+      await takeSnapshot('KLogo - rectangular background', snapshotOptions);
+    });
+    it('Render a logo with dimensions', async () => {
+      await renderComponentForVisualTest('KLogo', {
+        altText: 'Kolibri rectangular background',
+        size: '150',
+        backgroundStyle: 'rect',
+        showBackground: true,
+        maxSize: '60px',
+        minSize: '30px',
+      });
+      await takeSnapshot('KLogo - dimensions', snapshotOptions);
+    });
+  });
+
+  describe('Render a logo with different color schemes', () => {
+    it('color: whiteGrey', async () => {
+      await renderComponentForVisualTest('KLogo', {
+        altText: 'Kolibri logo - whiteGrey',
+        size: '150',
```

Comment:
I am not commenting on everything, just one more example to demonstrate the cleanup of unnecessary props I'd like to see - `size` and `backgroundStyle` is not needed to check color schemes. Please revisit all tests in the light of this.